config.harness

```diff
@@ -0,0 +1,15 @@
+export ALGOD_CHANNEL="$ALGOD_CHANNEL"
```

Comment:
env vars are substituted in `up.sh` before starting the sandbox with `/sandbox up harness`